test/integration/vendor/github.com/nginx/agent/sdk/v2/config_helpers.go

```diff
@@ -268,7 +268,7 @@ func updateNginxConfigFileConfig(
 				if err := updateNginxConfigFileWithRoot(aux, directive.Args[0], seen, allowedDirectories, directoryMap); err != nil {
 					return true, err
 				}
-			case "ssl_certificate", "proxy_ssl_certificate", "ssl_client_certificate", "ssl_trusted_certificate":
+			case "ssl_certificate", "proxy_ssl_certificate", "ssl_client_certificate", "ssl_trusted_certificate", "ssl_certificate_key":
```

Comment:
These vendor dirs are not up to date. 

Can you run `make deps` again?

Reply:
Hello,
Yes, sure I'm going to do it. 
Let me check to understand why it happened, to exclude such behavior in the future